### PR TITLE
Update read file content function

### PR DESCRIPTION
**Pull request by Dmitriy-NK**
Update file content read. Combine patch file reading and full-text file reading

### CODE REVIEW

File: credsweeper/file_handler/text_content_provider.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
+from os import read
 from typing import Dict, List, Optional
 
 from credsweeper.file_handler.analysis_target import AnalysisTarget
 from credsweeper.file_handler.content_provider import ContentProvider
-
+from credsweeper.utils import Util
 
 class TextContentProvider(ContentProvider):
```
Comment (meanrin) [inline credsweeper/file_handler/text_content_provider.py:8]: I think there are should be 2 empty lines between class and import, please check

`PEP 8: E302 expected 2 blank lines, found 1`

File: tests/file_handler/test_text_content_provider.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 from pathlib import Path
+from unittest import mock
```
Comment (meanrin) [inline tests/file_handler/test_text_content_provider.py:2]: `mock` is not used in this file

**Review by ARKAD97 — APPROVED** [review]
LGTM!

**Review by meanrin — APPROVED** [review]
This PR is similar to the #36 but for non-diff files
LGTM

**Pull request merged**